core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -48,6 +49,12 @@
 
     private static final Logger logger = LoggerFactory.getLogger(RetryingClient.class);
 
+    /**
+     * An Armeria specific header. This header is used to represent the total attempts.
+     * The server might use this value to reject excessive retries, etc.
+     */
+    public static final AsciiString TOTAL_ATTEMPTS = AsciiString.cached("total-attempts");
```

Comment:
Not sure this suits for this context. We should name it like `retry-count` to clearly indicate to the server?

Reply:
`retry-count` sounds better to me. Should we use `armeria-retry-count` or just `retry-count`?

Reply:
- `HttpHeaderNames.of(...)`
- First sentence is too vague. Could you reorganize the Javadoc?

Reply:
`armeria-retry-count` sounds better to me. let me use it with the value which is `totalAttempts - 1`. Thanks!